packages/dd-trace/test/appsec/iast/analyzers/hsts-header-missing-analyzer.spec.js

```diff
@@ -81,6 +81,24 @@ describe('hsts header missing analyzer', () => {
         expect(vulnerabilities[0].hash).to.be.equals(analyzer._createHash('HSTS_HEADER_MISSING:mocha'))
       }, makeRequestWithXFordwardedProtoHeader)
 
+      testThatRequestHasVulnerability((req, res) => {
+        res.setHeader('content-type', ['text/html'])
+        res.setHeader('Strict-Transport-Security', [])
+        res.end('<html><body><h1>Test</h1></body></html>')
+      }, HSTS_HEADER_MISSING, 1, function (vulnerabilities) {
+        expect(vulnerabilities[0].evidence.value).to.be.deep.equal([])
+        expect(vulnerabilities[0].hash).to.be.equals(analyzer._createHash('HSTS_HEADER_MISSING:mocha'))
+      }, makeRequestWithXFordwardedProtoHeader)
+
+      testThatRequestHasVulnerability((req, res) => {
+        res.setHeader('content-type', ['text/html'])
+        res.setHeader('Strict-Transport-Security', ['invalid1', 'invalid2'])
+        res.end('<html><body><h1>Test</h1></body></html>')
+      }, HSTS_HEADER_MISSING, 1, function (vulnerabilities) {
+        expect(vulnerabilities[0].evidence.value).to.be.deep.equal(['invalid1', 'invalid2'])
```

Comment:
What we should do with the [evidence resolution](https://github.com/DataDog/dd-trace-js/pull/3751/files#diff-7898738aed765e3d9dafa59774bec49e7fbd2052ea0714d2470857849136df54L43-L45) when we have detected a vulnerability in a header with array value?
res.getHeader() will return an array and it will be assigned as the evidence value...

Reply:
done